packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -234,13 +235,15 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
         :
         <Callout
           isBeakVisible={ false }
-          className={ css('ms-Dropdown-callout', styles.callout) }
+          className={ css('ms-Dropdown-callout', styles.callout, calloutProps ? calloutProps.className : undefined) }
+          data-automation-id='dropdown-callout'
           gapSpace={ 0 }
           doNotLayer={ false }
           targetElement={ this._dropDown }
           directionalHint={ DirectionalHint.bottomLeftEdge }
           onDismiss={ this._onDismiss }
           onPositioned={ this._onPositioned }
+          { ...calloutProps }
```

Comment:
hmmm I don't think this is what you want. This should go first. Then you should mix your stuff on top of it.